css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,44 @@
+h1 {
+  text-align: center;
+  font-family: "DM sans", sans-serif;
+}
+
+p {
+  font-family: "DM Sans", sans-serif;
+}
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  padding-left: 80px;
+  padding-right: 80px;
+}
+
+.one-five {
+  flex-basis: 20%;
+  margin: 10px;
+}
```

Comment:
Make this into a proper `column ` class. The `margin` should be applied to a separate rule for a separate class that you can add in addition to the `column` class, `column` classes should only have `flex-basis` or `width` applied to them.